openhtf/io/frontend/__main__.py

```diff
@@ -96,47 +96,41 @@ class StationStore(threading.Thread):
   """
   def __init__(self):
     super(StationStore, self).__init__()
-    self._death_event = threading.Event()
+    self._stop_event = threading.Event()
     self.hostname = socket.gethostname()
     self.stations = collections.defaultdict(Station)
```

Comment:
sadly, the semantics of `defaultdict` don't allow for passing args to the constructor.  I would make this a normal dict and add a method to the `StationStore` class to allow for 'lookup or create' semantics.  If you want to be fancy, you could even make it `__getitem__`, and then you get dict-like access to the `StationStore` directly:
```
def __getitem__(self, hostport):
  if hostport in self.stations:
    return self.stations[hostport]
  self.stations[hostport] = Station(hostport)
  return self.stations[hostport]
```
then down in _Track:
```
def _Track(self, *hostport):
  station = self[hostport]
  ...
```
And you can remove the `station.hostport = (host, port)` line.

The value here is defaultdict has potential to do some weird stuff with create-but-not-initialized Station instances.